deployment/edge/ansible/README.md

```diff
@@ -0,0 +1,19 @@
+Before launching ansible export a env to specify your local vio directory
+```
+export LOCAL_VIO_DIR=
+```
+
+
+### Install sshpass
+https://stackoverflow.com/questions/42835626/ansible-to-use-the-ssh-connection-type-with-passwords-you-must-install-the-s
+
+```
+pip install -r requirements.txt
+```
+
+
+Before launching Ansible for the first time 
+Be sure to launch ssh to have ip in known_hosts
+
+
+Scp du model doit tout supprimer pas en rajoute
```

Comment:
to remove

Reply:
Add explanation for 

      - /dev/video0:/dev/video0
      - /dev/video2:/dev/video2